manifests/config.pp

```diff
@@ -37,4 +37,18 @@
     mode   => '0750',
   }
 
+  file { $pulpcore::allowed_import_path:
```

Comment:
Occurs, to me maybe we should have called this `allowed_import_paths` since its an array

Reply:
Agree, that's been annoying me this whole time.

Reply:
(I just kept it because I'm not confident enough in my puppet knowledge to know what the consequences would be)

Reply:
That's correct - changing it would be a breaking change now. You would need to update all modules that use the parameter (which is puppet-foreman_proxy_content now). That would mean another release. It's a lot of churn.